tendrl-api.spec

```diff
@@ -0,0 +1,76 @@
+Name: tendrl-api
+Version: 0.0.1
+Release: 1%{?dist}
+Summary: Collection of tendrl api extensions
+Group: Development/Languages
+License: LGPLv2+
+URL: https://github.com/Tendrl/tendrl-api
+Source0: %{name}-%{version}.tar.gz
+BuildArch: noarch
+
+BuildRequires: ruby
+BuildRequires: systemd-units
+
+Requires: ruby = 2.0
+Requires: rubygem-sinatra
+Requires: rubygem-sinatra-contrib
+Requires: rubygem-activesupport
+Requires: rubygem-etcd
+Requires: rubygem-puma
+Requires: rubygem-sinatra-cross_origin
+
+%description
+Collection of tendrl api.
+
+%package doc
+Summary: Documentation for %{name}
+Group: Documentation
+Requires: %{name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+
+%package httpd
+Summary: Tendrl api httpd
+Requires: %{name} = %{version}-%{release}
+BuildArch: noarch
+Requires: httpd
+
+%description httpd
+Tendrl api httpd
+
+%prep
+%setup
+
+%install
+install -m 755 --directory $RPM_BUILD_ROOT%{_datadir}/%{name}
+install -m 755 --directory config $RPM_BUILD_ROOT%{_datadir}/%{name}
+install -m 755 --directory lib $RPM_BUILD_ROOT%{_datadir}/%{name}/lib
+install -Dm 0644 *.ru *.rb $RPM_BUILD_ROOT%{_datadir}/%{name}
+install -Dm 0644 tendrl-apid.service $RPM_BUILD_ROOT%{_unitdir}/tendrl-apid.service
+install -Dm 0644 config/etcd.sample.yml $RPM_BUILD_ROOT%{_datadir}/doc/tendrl/config/etcd.yml
```

Comment:
The sample file gets copied as etcd.yml somewhere under /etc/tendrl. @anivargi would be able to confirm the location of the same. In addition, ship the *.sample files in the config directory under the doc directory you have here. Update the %files section for the doc package accordingly.